README.md

```diff
@@ -1,41 +1,78 @@
 # Intro to Debugging
 
-## Overview
-In this lesson, we'll look at a few different ways to print data to the JavaScript console as a means of debugging our code.
+## Problem Statement
+As developers, one of the things that we spend about half of our time on is
+debugging. This is absolutely normal, and part of the development process. With
```

Comment:
Maybe give a brief definition of what `debugging` means

Reply:
....definitely helpful. :-)